src/schema/city/cityDataSortedByDisplayPreference.json

```diff
@@ -2,31 +2,55 @@
   {
     "slug": "new-york-ny-usa",
     "name": "New York",
-    "coordinates": { "lat": 40.71, "lng": -74.01 }
+    "coordinates": { "lat": 40.71, "lng": -74.01 },
+    "maxBounds": {
+      "sw": { "lat": "40.144", "lng": "-74.754" },
+      "ne": { "lat": "41.272", "lng": "-73.266" }
+    }
```

Comment:
Why are the new coordinates strings vs numbers? Or rather, I get that often you want to represent floats and large ints as strings in JSON, but then why are we leaving the epicenter coordinates to use numbers?

Reply:
Lol, bleary-eyed oversight bc I forgot `Number.toFixed` returns a string. Corrected now.